security/security-spring/src/main/resources/applicationContext-security.xml

```diff
@@ -57,6 +57,7 @@
     <!-- This must come before the default entry point which will capture everything not matched by pattern -->
     <http use-expressions="true" pattern="/api/**" entry-point-ref="restAuthenticationEntryPoint" create-session="never">
         <intercept-url pattern="/**" access="isAuthenticated()"/>
+        <csrf disabled="true"/>
```

Comment:
It was necessary to disable csrf requirements to be backwards compatible. We might want to think this through for the future ... and consider the potential risks of session hijacking.

Reply:
Yes, I agree.  If we enable csrf, we will have to change all front-end endpoints to post and start passing a csrf token.  Since our default behavior to this point has been disabled csrf, I figured we could leave this for another effort.